### PR TITLE
Switch from AutoParcel to AutoValue

### PR DESCRIPTION
**Pull request by gabrielittner**

### CODE REVIEW

File: sample/src/main/java/com/example/sqlbrite/todo/db/TodoItem.java [hunk 1]
```diff
@@ -17,10 +17,10 @@
 
 import android.content.ContentValues;
 import android.database.Cursor;
-import auto.parcel.AutoParcel;
+import com.google.auto.value.AutoValue;
 import rx.functions.Func1;
 
-@AutoParcel
+@AutoValue
 public abstract class TodoItem {
```
Comment (AlecStrong) [inline sample/src/main/java/com/example/sqlbrite/todo/db/TodoItem.java]: add `implements Parcelable`

File: sample/src/main/java/com/example/sqlbrite/todo/db/TodoList.java [hunk 2]
```diff
@@ -17,13 +17,13 @@
...
-import auto.parcel.AutoParcel;
+import com.google.auto.value.AutoValue;
 import java.util.ArrayList;
 import java.util.List;
 import rx.functions.Func1;
 
 // Note: normally I wouldn't prefix table classes but I didn't want 'List' to be overloaded.
-@AutoParcel
+@AutoValue
 public abstract class TodoList {
```
Comment (AlecStrong) [inline sample/src/main/java/com/example/sqlbrite/todo/db/TodoList.java]: same here

File: sample/src/main/java/com/example/sqlbrite/todo/ui/ListsItem.java [hunk 3]
```diff
@@ -16,15 +16,15 @@
...
 import com.example.sqlbrite.todo.db.TodoItem;
 import com.example.sqlbrite.todo.db.TodoList;
+import com.google.auto.value.AutoValue;
 import java.util.Arrays;
 import java.util.Collection;
 import rx.functions.Func1;
 
-@AutoParcel
+@AutoValue
 abstract class ListsItem {
```
Comment (AlecStrong) [inline sample/src/main/java/com/example/sqlbrite/todo/ui/ListsItem.java]: and here

**Comment by gabrielittner** [comment]
done

**Comment by AlecStrong** [comment]
LGTM

**Comment by JakeWharton** [comment]
Looking into lint failure.

**Comment by gabrielittner** [comment]
I guess it's happening because of including auto-value as provided. So I could either just copy the annotation in the sample or disable the lint check for the sample.

**Comment by JakeWharton** [comment]
yep. i got it: https://github.com/square/sqlbrite/pull/104

**Pull request closed without merging**